content/blog/greenlake-platform-infrastructure-monitoring-with-apache-skywalking.md

```diff
@@ -0,0 +1,144 @@
+---
+title: GreenLake Platform Infrastructure monitoring with Apache SkyWalking
+date: 2022-12-13T08:52:15.545Z
+author: Thirukkannan M
+authorimage: /img/thirupp_192x192.jpg
+thumbnailimage: /img/thirupp_192x192.jpg
+disable: false
+tags:
+  - Infrastructure Monitoring
+  - Apache SkyWalking
+---
+**Overview**
+
+HPE GreenLake Central is an advanced software-as-a-service platform that provides you with a consistent cloud experience for all your applications and data - on-premises or off-premises. It provides you with insights and controls to manage your hybrid IT estate, complementing your use of public clouds and data centers. HPE GreenLake Central gives you the ability to choose where and how to place your workloads and data, and - through the services you purchase - enables you to monitor security, capacity, resource utilization, and costs.
+
+HPE GreenLake for Private Cloud Enterprise is a managed, pay-per-use, Private Cloud as a Service (PCaaS) solution that provides a self-service environment where you can implement a hybrid approach to run different kinds of bare metal, virtual machine, and container workloads. These workloads run from a single site without the need to create dedicated isolated environments and manage the underlying infrastructure, resulting in more flexibility and better ROI with the least operational overhead.
+
+SkyWalking is an open source observability platform used to collect, analyze, aggregate and visualize data from services and cloud native infrastructures. SkyWalking provides an easy way to maintain a clear view of your distributed systems, even across Clouds. It is a modern APM (Application performance Management), specially designed for cloud native, container based distributed systems.
+
+**Monitoring** is tooling or a technical solution that allows teams to watch and understand the state of their systems. Monitoring is based on gathering predefined sets of metrics or logs.
+
+**Observability** is tooling or a technical solution that allows teams to actively debug their system. Observability is based on exploring properties and patterns not defined in advance.
+
+SkyWalking could be used for monitoring and observability. In this blog post we would focus on how we could setup monitoring of infrastructure (Virtual Machine and/or Bare metal) instances using popular APM tool like SkyWalking.
+
+**Pre-requisite**
+
+* An active service subscription to HPE GreenLake Private Cloud Enterprise.
+* VM and/or Bare metal instances are provisioned for the workloads and we are able to login to console.
+* SkyWalking instance has been setup either on premises or in Hyperscalers. Ensure that inbound ingress traffic is allowed on port 11800 for SkyWalking instance.
+
+ **Steps to configure monitoring**
+
+1. Monitoring host metric with node exporter
```

Comment:
This may be what Denis is referring to... when you look at this in Preview form, number 1 of the bullets appears to be in a different sized font than numbers 2 and 3...

![image](https://user-images.githubusercontent.com/77496261/207458909-15b1b328-dce6-4b0c-9af2-bfc124c6dbbe.png)